ci/run_artifacts.ps1

```diff
@@ -0,0 +1,17 @@
+[xml]$OldXML = Get-Content "/home/vsts/work/artifacts/Code Coverage Report_*/summary*/coverage.xml"
+[xml]$NewXML = Get-Content .\coverage.xml
+
+$oldlinerate = $OldXML.coverage.'line-rate'
+$newlinerate = $NewXML.coverage.'line-rate'
+
+$oldoutput = -join("Old line coverage rate: ", $oldlinerate)
+$newoutput = -join("New line coverage rate: ", $newlinerate)
+
+Write-Output $oldoutput
```

Comment:
You don't need the `-join` above I think, you could just do:
```powershell
Write-Host "Master line coverage: " $oldlinerate
```

With Write-Host, you can even set the foreground/background color to red/green if it is higher and lower, to make it extra fancy :)

Trivia of the day: In powershell, `Write-Output` (equivalent to `echo` in bash) writes to stdout, meaning if you piped the script into to a file i.e. `./run_artifacts.ps > out.txt`, write-output text would be in that file. `Write-Host` on the other hand, only outputs to the console (for humans to read) not to a stdout. So if you ran `./run_artifacts.ps > out.txt`, output from Write-Host would STILL be printed in the console, and not go into the file (which is why you can add colors and formatting to write-host, it is meant for console output, not for standard output)

Also please change old/new to refer to the branch names instead.